docs/self-managed/zeebe-deployment/configuration/logging.md

```diff
@@ -3,65 +3,52 @@ id: logging
 title: "Logging"
 ---
 
-Zeebe uses Log4j2 framework for logging. In the distribution and the Docker image, find the default log configuration file in `config/log4j2.xml`.
+The Camunda 8 orchestration cluster uses Log4j2 framework for logging. In the distribution and the Docker image, find the default log configuration file
+in `config/log4j2.xml`.
 
 ## Google Stackdriver (JSON) logging
 
 To enable Google Stackdriver compatible JSON logging, set the environment variable `ZEEBE_LOG_APPENDER=Stackdriver` before starting Zeebe.
```

Comment:
With Stackdriver now explained in the next section, I'm wondering if this could be removed?